pkgs/applications/system/asusctl/default.nix

```diff
@@ -0,0 +1,102 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, e2fsprogs
+, fontconfig
+, libGL
+, libglvnd
+, pkg-config
+, rustPlatform
+, udev
+, xorg
+, xwayland
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "asusctl";
+  version = "4.5.0";
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    udev
+    fontconfig
+    xwayland
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    e2fsprogs
+    libGL
+    libglvnd
+    systemd
+  ];
+
+  src = fetchFromGitLab {
```

Comment:
src is normally put between version and nativeBuildInputs

Reply:
Done